erpnext/stock/report/stock_ageing/stock_ageing.py

```diff
@@ -177,7 +177,7 @@ def get_fifo_queue(filters, sle=None):
 			else:
 				qty_to_pop = abs(d.actual_qty)
 				while qty_to_pop:
-					batch = fifo_queue[0] if fifo_queue else [0, None]
+					batch = fifo_queue[0] if fifo_queue else [0, d.posting_date]
```

Comment:
@rohitwaghchaure None is intentional for items with zero actual_qty. The None values can be filtered in the stock balance report itself like in stock ageing report 
https://github.com/frappe/erpnext/blob/305ec78c93305ed907803444cfb114c7e06bae98/erpnext/stock/report/stock_ageing/stock_ageing.py#L16
https://github.com/frappe/erpnext/blob/305ec78c93305ed907803444cfb114c7e06bae98/erpnext/stock/report/stock_ageing/stock_ageing.py#L21